pkg/langserver/autotype_win.go

```diff
@@ -20,6 +20,12 @@ var (
 		Modifiers: win32.ModCtrl,
 		KeyCode:   'I',
 	}
+	// AutotypeSSCHotkey is the hotkey to trigger auto-typing when in the SSC
+	AutotypeSSCHotkey = &win32.Hotkey{
+		ID:        1,
+		Modifiers: win32.ModCtrl,
+		KeyCode:   'U',
```

Comment:
What about the other autotype-commands? (Ctrl+P and Ctrl+O). Do these work in the SSC, or do they also require modifications?

If they also need their own SSC-Hotkey, that would result in 6 hotkeys in total, which is probably hard to remember.
How about making the SSC-Hotkeys a variant of the normal hotkeys?
Like normal insert is Ctrl+I and SSC-Insert is Ctrl+Alt+I ?
That would be done via ```Modifiers: win32.ModCtrl | win32.ModAlt``` and a keyup event for the Alt-Key (like in line 80)

Reply:
Clearing a chip isn't as important in the SSC, because you can just place a new chip (no cost). And given the option between insert and overwrite, I chose the command that was more useful (overwrite works in an empty chip, but insert doesn't work in a full chip).

That way there was only one key to bind, since there's no way for the script to know the context that the user is in, and four keys is already a lot for causing potential conflicts with the user's keybinds.